plugin/src/main/java/com/denizenscript/denizen/utilities/MultiVersionHelper1_20.java

```diff
@@ -0,0 +1,42 @@
+package com.denizenscript.denizen.utilities;
+
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.utilities.EnumHelper;
+import org.bukkit.Material;
+import org.bukkit.block.Sign;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.objects.core.MapTag;
+import org.bukkit.block.sign.Side;
+
+public class MultiVersionHelper1_20 {
+    public static boolean isSide(String side) {
+        return side != null && EnumHelper.get(Side.class).valuesMapLower.containsKey(EnumHelper.cleanKey(side));
+    }

```

Comment:
`EnumHelper` is not really meant to be used directly like that - although since this is only ever used for the `sign` command's input checking and it's just 2 values, I'd probably just manually compare it there/create a copypasta `Side` enum for input checking.